src/stylesheets/components/_banner.scss

```diff
@@ -141,6 +141,12 @@ $banner-icon-close: (
     middle,
     $theme-banner-background-color
   );
+
+  &::after {
```

Comment:
Pseudo element has media query. On line `247` it's the opposite. Please use consistent style.